helmfile.d/nginx-ingress.yaml

```diff
@@ -37,6 +37,10 @@ releases:
             # I doubt we need it at the moment, hence this comment.
             # https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#use-geoip2
             use-geoip2: "true"
+            hsts: "true"
+            hsts-preload: "true"
+            hsts-include-subdomains: "true"
```

Comment:
Enabling HSTS by default on every website under jenkins.io and jenkins-ci.org for two years is seriously annoying. We've been there and decided to only enable it for specific endpoints.
Last time we broke the mirroring infrastructure with that setting which should be fine now we it's available on https but we need to double check

Reply:
I think that by default I would enable HSTS by default *but* I would not enable it for subdomains so we can still disable HSTS for a specific website if we need it

Reply:
I _think_ it should be ok, but it will probably break something =/

Reply:
see comments below